builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -237,6 +237,7 @@ def loadServiceConfigurationData() {
 
     if ((service_name.trim() == "jazz_logs") || (service_name.trim() == "jazz_cloud-logs-streamer") || (service_name.trim() == "jazz_es-kinesis-log-streamer")) {
       updateConfigValue("{inst_elastic_search_hostname}", config_loader.JAZZ.ES_HOSTNAME)
```

Comment:
Will this line go away now that we have `config_loader.JAZZ.KIBANA_HOSTNAME`?

Reply:
We need ES_HOSTNAME to push data.